src/components/Movies.jsx

```diff
@@ -0,0 +1,22 @@
+import Movie from './Movie'
+import '../styles/movies.scss'
+
+const Movies = ({ movies, viewTrailer, closeCard }) => {
```

Comment:
To follow the Single Responsibility Principle (SRP), the logic for mapping movies should be separated into its own component
Create a MovieList.jsx component
Keep the Movies component as a container that organizes these components